source/include/umlalid.f90

```diff
@@ -93,34 +94,34 @@
   call dapek(damap(ii),jj,au(i3,i3))
   jj(i3)=0
 
-!     Store tas matrix (normalisation of phase space) and closed orbit for FMA and DUMP normalization.
-!     Variable added to DUMP block module variables;
-!     units dumptas: mm,mrad,mm,mrad,mm,1.e-3 -> convert later to 1.e3
-  if(ic(i)-nblo.gt.0) then !check if structure element is a block
-    if(ldump(ic(i)-nblo)) then !check if particles are dumped at this element
-      dumptas(ic(i)-nblo,ii-1,ii-1)=angp(1,ii-1)
-      dumptas(ic(i)-nblo,ii-1,ii  )=angp(1,ii)
-      dumptas(ic(i)-nblo,ii  ,ii-1)=au(ii,ii-1)
-      dumptas(ic(i)-nblo,ii  ,ii  )=au(ii,ii  )
-      dumptas(ic(i)-nblo,ii-1,i2-1)=au(i2-1,i2-1)
-      dumptas(ic(i)-nblo,ii  ,i2-1)=au(i2  ,i2-1)
-      dumptas(ic(i)-nblo,ii-1,i2  )=au(i2-1,i2  )
-      dumptas(ic(i)-nblo,ii  ,i2  )=au(i2  ,i2  )
-      dumptas(ic(i)-nblo,ii-1,i3-1)=au(i3-1,i3-1)
-      dumptas(ic(i)-nblo,ii  ,i3-1)=au(i3  ,i3-1)
-      dumptas(ic(i)-nblo,ii-1,i3  )=au(i3-1,i3  )
-      dumptas(ic(i)-nblo,ii  ,i3  )=au(i3  ,i3  )
-!    closed orbit in canonical variables x,px,y,py,sig,delta [mm,mrad,mm,mrad,mm,1.e-3]
-!    convert to x,xp,y,yp,sig,delta [mm,mrad,mm,mrad,mm,1]
-!     -> check units used in dumpclo (is x' or px used?)
-      dumpclo(ic(i)-nblo,2*j-1)=c(j)
-      if (j.eq.3) then !dp/p
-        dumpclo(ic(i)-nblo,2*j)  =cp(j)*c1m3
+  ! Store tas matrix (normalisation of phase space) and closed orbit for FMA and DUMP normalization.
+  ! Variable added to DUMP block module variables;
+  ! Units tasData: mm,mrad,mm,mrad,mm,1.e-3 -> convert later to 1.e3
```

Comment:
Is this converted to 1e3 or 1e0?

Reply:
???

Reply:
See line 161 to 164

Reply:
Just deleted the two comment lines as the information is duplicated anyway. Corrected the comment further down.